src/Shaders/particles.vertex.fx

```diff
@@ -2,10 +2,12 @@
 attribute vec3 position;
 attribute vec4 color;
 attribute vec4 options;
+attribute vec4 cellInfo;

```

Comment:
As I mentioned before the cellInfo attribute should only be a float
and we must have an uniform with the cell size info